### DISCUSSION TITLE
Would this in theory save battery over emulation?

### DISCUSSION THREAD

**Discussion by deanhuff** [profi200/open_agb_firm#114] [Q&A]
Title sums up my question, would running the native process use less battery than emulation such as mGBA?

Bonus question, is there a way to toggle video between top & bottom screens (and change interpolation settings) on the fly through some shortcut?

**Comment by Masamune3210** [comment]
Emulation is almost always going to cause more power usage. I don't know if anyone has ever put in the effort to quantify it on the 3DS, but its something you could do yourself if you really wanted to. Just charge the battery to full, take it off the charger and time how long it takes for it to die, once while running a game through agb_firm and once through mgba

**Comment by profi200** [comment]
Mentioned it somewhere else already. I don't know how much battery this eats over emulation. It's probably a bit more efficient. If someone wants to run tests that would be nice.
